cmd/puppeth/wizard_intro.go

```diff
@@ -30,15 +30,18 @@ import (
 )
 
 // makeWizard creates and returns a new puppeth wizard.
-func makeWizard(network string, images map[string]string) *wizard {
+func makeWizard(network string, images map[string]string, bootnodes []string) *wizard {
 	return &wizard{
 		network: network,
 		conf: config{
 			Servers: make(map[string][]byte),
+			bootnodes: bootnodes,
+			static_bootnode: len(bootnodes),
```

Comment:
If `static_bootnode` means length of `bootnodes`, why don't you just use `len(cfg.bootnodes)`, instead of `cfg.static_bootnode`?

Reply:
yes It is look good, but it makes bootnode keep appending.
cause `networkState()` 
https://github.com/Onther-Tech/plasma-evm/blob/237a037c9c3d0c63b9ee823cdc1d9712839dbcf1/cmd/puppeth/wizard_netstats.go#L38-L40

so, The number of bootnodes from flag should be fixed variable. 👍